build/NuSpecs/tools/Web.config.install.xdt

```diff
@@ -181,15 +181,35 @@
 
   <location path="umbraco" xdt:Locator="Match(path)" xdt:Transform="InsertIfMissing" />
   <location path="umbraco" xdt:Locator="Match(path)">
-    <system.webServer xdt:Transform="InsertIfMissing">
+    <system.webServer xdt:Transform="InsertIfMissing" />
+  </location>
+  <location path="umbraco" xdt:Locator="Match(path)">
+    <system.webServer>
+      <urlCompression xdt:Transform="InsertIfMissing" />
+      <httpErrors xdt:Transform="InsertIfMissing" />
+    </system.webServer>
+  </location>
+  <location path="umbraco" xdt:Locator="Match(path)">
+    <system.webServer>
       <urlCompression doStaticCompression="false" doDynamicCompression="false" dynamicCompressionBeforeCache="false" xdt:Transform="SetAttributes(doStaticCompression,doDynamicCompression,dynamicCompressionBeforeCache)" />
+      <httpErrors errorMode="Custom" existingResponse="PassThrough" xdt:Transform="SetAttributes(errorMode,existingResponse)"/>
```

Comment:
```suggestion
      <httpErrors existingResponse="Auto" xdt:Transform="SetAttributes" />
```
There's no need to repeat the attribute names, as it will only set attributes that are present on the element: https://docs.microsoft.com/en-us/previous-versions/aspnet/dd465326(v=vs.110)?redirectedfrom=MSDN#setattributes